src/backend/src/app-mux.jl

```diff
@@ -11,7 +11,7 @@ using TOML
 
 
 #const DATA_ROOT = joinpath(@__DIR__, "../data/")
-const DATA_ROOT = get(ENV, "ITO_OD_DATA_ROOT", joinpath(@__DIR__, "../testdata/processed"))
+const DATA_ROOT = get(ENV, "ITP_OD_DATA_ROOT", joinpath(@__DIR__, "../testdata/processed"))
```

Comment:
:facepalm: sorry! 

I don't know how I missed this - I checked it at least three times to make sure it was right when I typed it out in the Merlin backend earlier today...